coderd/coderd.go

```diff
@@ -415,6 +415,7 @@ func New(options *Options) *API {
 				r.Use(httpmw.ExtractWorkspaceAgent(options.Database))
 				r.Get("/metadata", api.workspaceAgentMetadata)
 				r.Post("/version", api.postWorkspaceAgentVersion)
+				r.Post("/app-healths", api.postWorkspaceAppHealths)
```

Comment:
I'd say `health` would be representative of the plural in this case too. `/app-health` sounds a bit better to me.